config/default/block.block.subsitebanner.yml

```diff
@@ -9,8 +9,8 @@ dependencies:
     - essex
 id: subsitebanner
 theme: essex
-region: banner
```

Comment:
I wonder if this block being called 'Subsite banner' and being place in the 'hero' region might eventually seem confusing in the future?

Reply:
Yeah its a good point, I had initial thoughts about re-doing the regions that I had created once I finished. The banner region is no longer needed as its mainly the banner_content region which I am using. So maybe I leave it as just the banner content region